src/lib/button-toggle/button-toggle.md

```diff
@@ -18,5 +18,12 @@ and `ngModel` is not supported.
 The button-toggles will present themselves as either checkboxes or radio-buttons based on the 
 presence of the `multiple` attribute. 
 
+For button toggles with icons only, each button toggle should be marked with a meaningful label by 
+`aria-label` or `aria-labelledby`.
+
+For button toggle groups, each group should be marked with a meaningful label by adding `aria-label`
```

Comment:
I would rephrase ``"marked with a meaningful label by adding `aria-label` or `aria-labelledby"`` to 
``"given a meaningful label via `aria-label` or `aria-labelledby`"`` here and elsewhere